<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer::Internal::ImageViewerToolbar</class>
 <widget class="QWidget" name="ImageViewer::Internal::ImageViewerToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>28</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="Core::CommandButton" name="toolButtonBackground">
     <property name="toolTipBase">
      <string>Show Background</string>
     </property>
     <property name="icon">
      <iconset resource="imageviewer.qrc">
       <normaloff>:/imageviewer/images/background.png</normaloff>:/imageviewer/images/background.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonOutline">
     <property name="toolTipBase">
      <string>Show Outline</string>
     </property>
     <property name="icon">
      <iconset resource="imageviewer.qrc">
       <normaloff>:/imageviewer/images/outline.png</normaloff>:/imageviewer/images/outline.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonFitToScreen">
     <property name="toolTipBase">
      <string>Fit to Screen</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonOriginalSize">
     <property name="toolTipBase">
      <string>Original Size</string>
     </property>
     <property name="icon">
      <iconset resource="imageviewer.qrc">
       <normaloff>:/imageviewer/images/originalsize.png</normaloff>:/imageviewer/images/originalsize.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonZoomIn">
     <property name="toolTipBase">
      <string>Zoom In</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonZoomOut">
     <property name="toolTipBase">
      <string>Zoom Out</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Core::CommandButton" name="toolButtonPlayPause">
    </widget>
   </item>
   <item>
    <widget class="Utils::StyledSeparator" name="widgetToolbarSeparator" native="true"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>315</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Utils::StyledSeparator" name="widgetSizeSeparator" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="labelImageSize"/>
   </item>
   <item>
    <widget class="Utils::StyledSeparator" name="widgetInfoSeparator" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="labelInfo"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::StyledSeparator</class>
   <extends>QWidget</extends>
   <header>utils/styledbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Core::CommandButton</class>
   <extends>QToolButton</extends>
   <header>coreplugin/actionmanager/commandbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imageviewer.qrc"/>
 </resources>
 <connections/>
</ui>
